src/listfunc.c

```diff
@@ -1692,32 +1694,32 @@ Obj FuncCOPY_LIST_ENTRIES( Obj self, Obj args )
   else
     {
       Obj tmplist = NEW_PLIST(T_PLIST,number);
-      Obj *tptr = ADDR_OBJ(tmplist)+1;
```

Comment:
While I don't want you to change it back, when doing things like this in future, consider leaving tptr. It wasn't doing any harm, and made convincing myself the diff was right 100x harder!

Reply:
Good point, and in fact, looking at this diff, I am flabbergasted as to why I changed it... Huh?